plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImagePage.java

```diff
@@ -225,7 +227,14 @@ public IStatus validate(Object value) {
 			.observe(model))
 			.in(dbc);
 		ControlDecorationSupport.create(
-			selectedConnectionBinding, SWT.LEFT | SWT.TOP, null, new RequiredControlDecorationUpdater(true));	}
+			selectedConnectionBinding, SWT.LEFT | SWT.TOP, null, new RequiredControlDecorationUpdater(true));
+		
+        StyledText newDockerConnectionLink = StyledTextUtils.emulateLinkWidget("<a>New Docker connection</a>", new StyledText(parent, SWT.WRAP));
```

Comment:
Then, I would rather suggest button "Manage Projects..." for general consistency. A link to the right of text/combo input which allows to create a new item is not a common solution in Eclipse; buttons are common.